### PR TITLE
completed task 6

### PR DESCRIPTION
**Pull request by K-Othman**

### CODE REVIEW

File: models/news.js [hunk 1]
```diff
@@ -9,20 +9,42 @@ exports.selectTopics = () => {
...
+        });
+      }
+      return article;
+    });
+};
+
+exports.selectComment = (articleId) => {
+  return db
+    .query(
+      "SELECT comments.votes, comments.created_at, comments.author, comments.body, comments.article_id as comment_id FROM comments WHERE comments.article_id = $1 GROUP BY comments.votes, comments.created_at, comments.author, comments.body, comments.article_id ORDER BY created_at DESC;",
```
Comment (MRombi) [inline models/news.js:39]: We do not need to group by, please delete that part 🙂

File: models/news.js [hunk 2]
```diff
@@ -9,20 +9,42 @@ exports.selectTopics = () => {
...
+      [articleId]
+    )
+    .then(({ rows }) => {
+      const comments = rows;
+      return comments;
+    });
+};
+
+// if (comments.length === 0 && article) {
```
Comment (MRombi) [inline models/news.js:48]: We should delete commented out code 🙂

**Review by MRombi — APPROVED** [review]
Good job on this endpoint! Please, delete the bits I mentioned in the comments and merge 😄

File: models/news.js (hunk 1 above)
Comment (K-Othman) [reply models/news.js:39]: done thanks!

**Pull request merged**